test/linker.test.ts

```diff
@@ -464,6 +464,65 @@ describe('Wollok linker', () => {
       C.methods[0].sentences[0].should.target(C.fields[0])
     })
 
+    it('should target references to members inherited from superclass in different packages', () => {
+      const environment = link([
+        new Package({
+          name: 'aaa',
+          imports: [
+            new Import({ isGeneric: true, entity: new Reference({ name: 'bbb' }) }),
+          ],
+          members: [
+            new Class({
+              name: 'C',
+              supertypes: [new ParameterizedType({ reference: new Reference({ name: 'bbb.B' }) })],
```

Comment:
Esto debería hacer referencia a `B` solamente, el archivo ya lo tenés importado arriba.

```suggestion
              supertypes: [new ParameterizedType({ reference: new Reference({ name: 'B' }) })],
```